apps/mark-scan/pat-device-input/src/pin.rs

```diff
@@ -51,6 +72,9 @@ impl Pin {
         self.set_direction_in()
     }
 
+    /// Reads the pin value and returns the boolean inverse of the pin's raw value.
+    /// # Example
+    /// When the pin's value is b'0', `is_active` returns true.
```

Comment:
I believe this convention is specific to VSAP's implementation, so it could arguably live in the daemon code. But since we have no planned use for this module outside the PAT integration it feels cleaner to leave this translation in the pin module.